hawc/apps/animal/templates/animal/endpoint_detail.html

```diff
@@ -1,42 +1,39 @@
 {% extends 'assessment-rooted.html' %}
 
 {% load hawc %}
+{% load bs4 %}
 
 {% block content %}
 
   {% if crud == "Read" %}
+  <div class="d-flex">
+    <h2>{{object}}</h2>
     {% if obj_perms.edit %}
-    <div class="dropdown btn-group float-right">
-      <a class="btn btn-primary dropdown-toggle" data-toggle="dropdown">Actions</a>
-      <div class="dropdown-menu dropdown-menu-right">
-        <span class="dropdown-header">Endpoint Editing</span>
-        <a class="dropdown-item" href="{{ object.get_update_url }}">Update endpoint</a>
-        <a class="dropdown-item" href="{% audit_url object %}">Change logs</a>
-        <a class="dropdown-item" href="{% url 'animal:endpoint_delete' object.pk %}">Delete endpoint</a>
-        {% if assessment.enable_bmd and object.bmd_modeling_possible %}
-          <div class="dropdown-divider"></div>
-          <span class="dropdown-header">BMD Modeling</span>
-          {% if bmd_session %}
-            <a class="dropdown-item" href="{{bmd_session.get_absolute_url}}">Latest session</a>
-            <a class="dropdown-item" href="{% url 'bmd:session_list' object.pk %}">All sessions</a>
-          {% elif assessment.bmd_settings.can_create_sessions %}
-            <a class="dropdown-item" href="{% url 'bmd:session_create' object.pk %}">Create</a>
+      {% actions %}
+          <span class="dropdown-header">Endpoint Editing</span>
+          <a class="dropdown-item" href="{{ object.get_update_url }}">Update endpoint</a>
+          <a class="dropdown-item" href="{% audit_url object %}">Change logs</a>
+          <a class="dropdown-item" href="{% url 'animal:endpoint_delete' object.pk %}">Delete endpoint</a>
+          {% if assessment.enable_bmd and object.bmd_modeling_possible %}
+            <div class="dropdown-divider"></div>
+            <span class="dropdown-header">BMD Modeling</span>
+            {% if bmd_session %}
+              <a class="dropdown-item" href="{{bmd_session.get_absolute_url}}">Latest session</a>
+              <a class="dropdown-item" href="{% url 'bmd:session_list' object.pk %}">All sessions</a>
+            {% elif assessment.bmd_settings.can_create_sessions %}
+              <a class="dropdown-item" href="{% url 'bmd:session_create' object.pk %}">Create</a>
+            {% endif %}
           {% endif %}
```

Comment:
this logic leads to this empty header section. can we either update the first if statement or show text on the empty case?
![image](https://github.com/shapiromatron/hawc/assets/42587248/ee186e36-3003-4dd4-b93d-46fd1ca591ba)

Reply:
@shapiromatron any preference on the above?

Reply:
I think it's better to switch up the if statement, so option 1, if you agree.